portfolio/src/main/webapp/style.css

```diff
@@ -7,3 +7,8 @@
 #greeting-container {
   margin-top: 20px;
 }
+
+.navbar-icon-background{
+    background-color: hsl(0, 0%, 96%);
+    border-color: hsl(0, 0%, 96%);
+}
```

Comment:
This file is missing a newline at the end of file, which is conventional.

Reply:
- I haven't started writing me.html, I just created for testing the logistic. Should I remove it before I actually write it?
- I might need to use other files in Bulma so I want to keep it for now maybe.
- I didn't start writing JS but I believe that it will mainly base on jQuery and not relate to Bulma.
- I think you should run it in ~/step/portfolio but that may also because there isn't any JS now in my webapp. I have updated the README.md

Reply:
I think the newline is still missing.

Reply:
Just kidding, I think I was looking at an old version.